Package.swift

```diff
@@ -6,6 +6,8 @@ let package = Package(
     name: "DittoSwiftTools",
     platforms: [
         .iOS(.v14),
+        .macOS(.v11),
```

Comment:
Can we remove macOS support again since its still broken? We've got a long term issue tracking adding it back